__tests__/app.test.js

```diff
@@ -18,3 +26,36 @@ describe.only("GET /api", () => {
   });
 });
 
+
+
+describe.only("GET /api/topics",()=>{
+  test("200: Responds with an array of topic objects where each topic object has properties of slug and description",()=>{
+    return request(app)
+    .get("/api/topics")
+    .then(({body:{topics}})=>{
+      expect(topics.length).toBe(3)
```

Comment:
Important to check the length in these kinds of tests, an empty array would pass the below expects inside the foreach!

Reply:
Hey, thanks for the feedback! Have i done this correctly then by checking the length, or do i need to correct something still?